### PR TITLE
data:shwetasng

### PR DESCRIPTION
**Pull request by shwetasng**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5603"></a>

### CODE REVIEW

File: data/shwetasng.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Shweta Singh",
+  "displayStatsPublic":true,
+  "type":"personal",
+  "bio":"student"
+  "avatar": "https://github.com/shwetasng.png"
```
Comment (Sindhuinti) [inline data/shwetasng.json:6]: It is not required to include your avatar, it will auto imported from github.

**Review by Sindhuinti — CHANGES_REQUESTED** [review]

File: data/shwetasng.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Shweta Singh",
+  "displayStatsPublic":true,
+  "type":"personal",
+  "bio":"student"
```
Comment (imbeshat) [inline data/shwetasng.json:5]:
```suggestion
  "bio":"student",
```

**Review by imbeshat — COMMENTED** [review]
Hey @shwetasng, just a suggestion you can also add links such as GitHub, LinkedIn, Twitter, etc.

File: data/shwetasng.json (hunk 1 above)
Comment (kumarsonsoff3) [reply data/shwetasng.json:6]:
```suggestion
```

File: data/shwetasng.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Shweta Singh",
+  "displayStatsPublic":true,
```
Comment (kumarsonsoff3) [inline data/shwetasng.json:3]: Also we don't need the `displayStatsPublic` key anymore.

```suggestion
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Comment by SaraJaoude** [comment]
You have conflicts in your file, please resolve. This is usually because your fork is not up to date.  

You may wish to have a look at the following Docs and video: https://linkfree.eddiehub.io/docs/how-to-guides/editing

**Review by iamthecloverly — APPROVED** [review]

File: data/shwetasng.json [hunk 4]
```diff
@@ -1,4 +1,4 @@
-{
+
```
Comment (SaraJaoude) [inline data/shwetasng.json:1]: this bracket is required for the Json to work

**Pull request closed without merging**